python/tests/test_host_redirect_errors.py

```diff
@@ -0,0 +1,179 @@
+from typing import Any, Dict, List, Tuple
```

Comment:
> Y'know, it's OK for tests to have comments, too. :slightly_smiling_face:

Reply:
..._something something_ hoist _something_ petard _something something_...

Yeah. You're right. I'll fix it soon. 🙂